api/src/main/java/com/okta/idx/sdk/api/client/IDXAuthenticationWrapper.java

```diff
@@ -279,11 +280,13 @@ public AuthenticationResponse register(ProceedContext proceedContext,
      * @return the Authentication response
      */
     public AuthenticationResponse selectAuthenticator(ProceedContext proceedContext,
-            com.okta.idx.sdk.api.client.Authenticator authenticator) {
+                                                      com.okta.idx.sdk.api.client.Authenticator authenticator) {
         try {
             return AuthenticationTransaction.proceed(client, proceedContext, () -> {
                 Authenticator authenticatorRequest = new Authenticator();
                 authenticatorRequest.setId(authenticator.getId());
+                authenticatorRequest.setEnrollmentId(authenticator.getFactors().get(0).getEnrollmentId());
```

Comment:
These shouldn't be called here. If you need these, you should use the `selectFactor` method.

Reply:
updated.